sections/clause_08c_rest_api_read.adoc

```diff
@@ -434,6 +434,7 @@ Note: Adapted from OData 4.01-Protocol 11.2.6.5
 ==== $orderby
 
 The `$orderby` system query option specifies the order in which items are returned from the service.
+If used with `$top` and `$skip`, the service SHALL apply `$orderby` before applying the other query options. 
```

Comment:
good point, but not here. I'll add a section of query option evaluation order, linking to file:///home/scf/ownCloud/Projects/SensorThingsApi/OData/odata-v4.01-part1-protocol.html#sec_SystemQueryOptions

Reply:
The document already contains the evaluation order: https://hylkevds.github.io/23-019/23-019.html#_evaluation_order